### ISSUE TITLE
How does it behaves with app bundles?

### ISSUE THREAD

**Issue by baneageorge** [Redth/ResizetizerNT#13]
Hi,

Great initiative. It is really necessary such a tool.
I am not sure I understood how it works but how does it behaves with app bundles?
I know the reason for those various sizes is also to pack the app in such a way to serve the assets relative to the screen size/resolution.and minimize the app size.

Does this tool maintain that functionality?

**Comment by Redth** [comment]
Since the generated resources are included just as any normal android drawable resource for all the different dpi's (eg: drawable-xhdpi, drawable-xxhdpi, etc), it works just as you would expect and hope it to with app bundles where only the resources for the applicable dpi get downloaded to the device from the play store.

tl;dr - it's going to do what you want/hope/expect just fine in app bundles
